app/views/layouts/application.html.erb

```diff
@@ -12,22 +12,28 @@
 
       <% if !user_signed_in? %>
         <%= link_to "Login", new_user_session_path %>  
+      <% else %>
+        <%= link_to "Logout", destroy_user_session_path, method: :delete %>
       <% end %>
-
-      <%= link_to "Logout", destroy_user_session_path, method: :delete %>
+      
       <%= link_to "Edit User", edit_user_registration_path %>
 
+      <%= link_to "Back", :back %>
+
     </div>
 
+
+    <p class="notice"><%= notice %></p>
```

Comment:
You should only be displaying these if notice/alert are non-empty

Reply:
Fixed this using an if and else if conditional statement.

Reply:
<% if notice %>
      <p class="notice"><%= notice %></p>
    <% end %>
    
    <% if alert %>
      <p class="alert"><%= alert %></p>
    <% end %>